microprofile/WFLY-14281_Integrate-MP-Fault-Tolerance-3.0.adoc

```diff
@@ -0,0 +1,86 @@
+= Integrate MP Fault Tolerance 3.0 into EAP
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+MicroProfile 4.0 provides a major upgrade for https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/3.0[MicroProfile Fault Tolerance 3.0] with changes described in its
+https://download.eclipse.org/microprofile/microprofile-fault-tolerance-3.0/microprofile-fault-tolerance-spec-3.0.html#release_notes_30[Release Notes].
+
+This component upgrade also covers the upgrade of SmallRye Fault Tolerance that implements MicroProfile Fault Tolerance 3.0.
+
+== Issue Metadata
+ 
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-14281[WFLY-14281 Integrate MP Fault Tolerance 3.0]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1594[EAP7-1594 Integrate MP Fault Tolerance 3.0 into EAP]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:rhusar@redhat.com[Radoslav Husar]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
```

Comment:
Can links be added to the list items here and below? It helps the reader browse to the exact content the writer has in mind when providing a reference..

Reply:
Link to what?

Reply:
Are the Quickstarts affected in any way by the upgrade?

Reply:
No (at least I haven't found anything that needs updating).